src/cascadia/LocalTests_SettingsModel/SerializationTests.cpp

```diff
@@ -0,0 +1,2373 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+
+#include "../TerminalSettingsModel/ColorScheme.h"
+#include "../TerminalSettingsModel/CascadiaSettings.h"
+#include "JsonTestClass.h"
+#include "TestUtils.h"
+#include <defaults.h>
+#include "../ut_app/TestDynamicProfileGenerator.h"
+
+using namespace Microsoft::Console;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+using namespace WEX::Common;
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
+
+namespace SettingsModelLocalTests
+{
+    // TODO:microsoft/terminal#3838:
+    // Unfortunately, these tests _WILL NOT_ work in our CI. We're waiting for
+    // an updated TAEF that will let us install framework packages when the test
+    // package is deployed. Until then, these tests won't deploy in CI.
+
+    class SerializationTests : public JsonTestClass

```

Comment:
these are Deserialization tests ...

Reply:
Fixed. Bear with me. This is one of those small things that I'll be hitting _a lot_ this month.